### PR TITLE
Proxy addition

### PR DESCRIPTION
**Pull request by rjkernick**

### CODE REVIEW

**Comment by rjkernick** [comment]
@alex1989hu Can you test this out and let me know if it resolves your issues?

fixes #14

**Comment by alex1989hu** [comment]
> @alex1989hu Can you test this out and let me know if it resolves your issues?
> 
> fixes #14

Sorry for being late. I am going to check tomorrow 👍

**Comment by alex1989hu** [comment]
I have just tested it, see the snippet below 👍
```bash
2020-03-20T16:35:39.346782583Z 2020-03-20 16:35:39 (9.23 MB/s) - 'sonar-scm-git-plugin-1.9.1.1834.jar' saved [3625116/3625116]
...
2020-03-20T16:35:40.671072363Z 2020-03-20 16:35:40 (2.50 MB/s) - 'sonar-java-plugin-6.2.0.21135.jar' saved [17071013/17071013]
```

**Comment by rjkernick** [comment]
Thanks @alex1989hu for testing that out. I'll get it merged in.

File: charts/sonarqube/templates/deployment.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ spec:
       {{- end }}
       {{- if .Values.plugins.install }}
         - name: install-plugins
-          image: {{ default "alpine:3.10.3" .Values.plugins.initContainerImage }}
+          image: {{ default "rjkernick/alpine-wget:latest" .Values.plugins.initContainerImage }}
```
Comment (usmonster) [inline charts/sonarqube/templates/deployment.yaml]: Thanks for the fix, but is this really required? Why not just use curl to grab the plugins, as suggested in `helm/charts#18805`? This means you don't have to maintain this one-off image, and we can keep the trusted official source as the default.

Comment (alex1989hu) [reply charts/sonarqube/templates/deployment.yaml]: @usmonster you are absolutely right. It is going to be tricky:  alpine:3.10.3 -> `/bin/sh: curl: not found`

Comment (pcaamano2000) [reply charts/sonarqube/templates/deployment.yaml]: Ah! I wasn't aware. 😞  Thanks for the quick response!

Comment (usmonster) [reply charts/sonarqube/templates/deployment.yaml]: Ah! I wasn't aware. 😞  Thanks for the quick response!

**Pull request merged**